README.md

```diff
@@ -11,47 +11,41 @@
+[mailto:IoTReady@lists.fiware.org](IoTReady@lists.fiware.org), who will review your tests logs and evaluate whether your
+device is connecting with the public instance correctly. Once you have successfully completed the FIWARE-ready IoT
+Devices Program, your device will be published on `http://marketplace.fiware.org`.
 
 | :books: [Documentation](https://fiware-ready-iot-devices-validation.readthedocs.io/en/latest/) |
 | ---------------------------------------------------------------------------------------------- |
 
 
 ## Background
 
-Over 120 cities in Europe and Latin America belong to the Open & Agile Smart Cities (OASC) Alliance Initiative. They
-have agreed to operate, share and publish their smart city data using the FIWARE interoperability model.
-
-The FIWARE ecosystem means a straightforward business opportunity for IoT products: It is simple, the FIWARE IoT stack
-handles well-known IoT protocol standards and provides an enhanced OpenStack-based cloud environment plus a powerful set
-of standard REST APIs to ease the development of smart applications.
+The FIWARE ecosystem means a straightforward business opportunity for IoT products: It is simple, the **FIWARE IoT
+stack** handles well-known IoT protocol standards and provides an enhanced OpenStack-based cloud environment plus a
+powerful set of standard REST APIs to ease the development of smart applications.
 
-Thousands of companies, including large corporations, startups, SMEs and individual developers worldwide are willing to
-provide their Internet of Things (IoT) solutions to these cities, basing their solutions on FIWARE open source platform
-[components](https://github.com/fiware/catalogue).
+A growing number of companies, including large corporations, startups, SMEs and individual developers worldwide are
+willing to provide their solutions based on FIWARE open-source [components](https://github.com/fiware/catalogue).
 
-So far, developers needed to search for **compatible** IoT products and find out how to make them work. The FIWARE-ready
-IoT Devices Program provides the answer as it is designed to demonstrate the ability of the IoT solutions to
-interoperate seamlessly with FIWARE platforms. The database of products and developers that have successfully
-accomplished the FIWARE-ready IoT Devices Program is the catalogue of the ecosystem of IoT hardware producers and
-technology providers that can be selected by the OASC Alliance cities to build their smart city solutions.
+So far, developers needed to search compatible IoT products and find out how to make them work. The certification of
+FIWARE-ready IoT Devices provides the answer as it is designed to demonstrate the ability of the IoT solutions to
+seamless interoperate with FIWARE platforms. The database of products and developers that have successfully been
+featured and certified as FIWARE-ready IoT Devices will be part of the FIWARE Marketplace, a global one-stop shop that
```

Comment:
certified idem

Reply:
Known issue, but I **won't fix** - at least not with this PR - this terminology is a separate issue for discussion. I'm just altering the intro for now.